### DISCUSSION TITLE
Traking drives dashboard and elevation

### DISCUSSION THREAD

**Discussion by djpalmis** [jheredianet/Teslamate-CustomGrafanaDashboards#24] [Q&A] [answered]
![Screenshot_20230205_205548_Samsung Internet](https://user-images.githubusercontent.com/402122/216841807-ed52be35-a993-4ccf-982f-d33ed720e448.jpg)
I  guess there is something wrong with the elevation information..
Am I misunderstanding the meaning or the computation is wrong?

**Comment by jheredianet** [comment, marked as answer]
Hi, Elevation is not a computation is a reading of what you have logged on Teslamate. In the chart what you have to see is the Y axis on the right. Could you compare if it is the same as your drive details (from TeslaMate dashboard). It's should be the same. It would be useful if you can share a screenshot of that dashboard too. 

In your case it says 283m up and -301m down. This is interpreted in the following way: during the entire journey, a sum is made of everything that has gone up and the sum of what has gone down. And as you can see in the graph, you have been on a road with many slopes going up and down. Your arrival elevation was 34m and initial elevation was 52m (more or less as shown in the graph), so you have a difference of in that trip of 283m up -301m down = -18m (descending), which is the same as 34m - 52m = -18m.

**Comment by djpalmis** [comment]
![Screenshot_20230206_155332_Samsung Internet](https://user-images.githubusercontent.com/402122/217004789-8741f3f2-4029-4056-84e8-b91669a7763a.jpg)
You're right: it's the same on original teslamate. 
So it is a sum of every step..  ok, that's why I didn't opened a ticket for a bug, because I was not sure about the meaning of that value.
Thank You

**Reply by jheredianet** [reply]
You got it. Happy to help! :-)
